core/designsystem/src/main/kotlin/com/dhc/designsystem/graph/DhcGraph.kt

```diff
@@ -0,0 +1,288 @@
+package com.dhc.designsystem.graph
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.layout
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import com.dhc.common.drawBalloonTail
+import com.dhc.designsystem.AccentColor
+import com.dhc.designsystem.DhcTheme
+import com.dhc.designsystem.DhcTypoTokens
+import com.dhc.designsystem.LocalDhcColors
+import com.dhc.designsystem.SurfaceColor
+import com.dhc.designsystem.graph.model.DhcGraphConfig
+import com.dhc.designsystem.graph.model.DhcGraphData
+
+@Composable
+fun DhcGraph(
+    graphData: List<DhcGraphData>,
+    dhcGraphConfig: DhcGraphConfig,
+    modifier: Modifier = Modifier,
+) {
+    val rowLabelStyle = DhcTypoTokens.Body6
+    val columnLabelStyle = DhcTypoTokens.TitleH7
+
+    ConstraintLayout(modifier = modifier) {
+        val (graphRowLabels, graphRowLines, graphBars, graphColumnLabels) = createRefs()
+
+        DhcGraphRowLabels(
+            modifier = Modifier
+                .padding(bottom = columnLabelStyle.fontSize.value.dp)
+                .fillMaxHeight()
+                .constrainAs(graphRowLabels) {
+                    top.linkTo(parent.top)
+                    start.linkTo(parent.start)
+                },
```

Comment:
위에 계원오빠가 말한거 컬럼을 맨위로 올리고 이렇게 하면될 것 같기도~~!!
```suggestion
.constrainAs(graphRowLabels) {
                    top.linkTo(parent.top)
                    start.linkTo(parent.start)
                    bottom.linkTo(graphColumnLabels.top)
                    height = Dimension.fillToConstraints
                },
```